app/src/main/java/UtilityClass/instructorData.java

```diff
@@ -0,0 +1,45 @@
+package UtilityClass;
+
+import java.util.HashMap;
+
+/**
+ * Created by chandx on 9/19/17.
+ */
+
+public class instructorData {
+    public HashMap<String, HashMap<String, String>> courses;
+    public String email;
+    public HashMap<String, HashMap<String, String>> reviews;
+
+    public instructorData(){}
+
+    public instructorData(HashMap<String, HashMap<String, String>> inputData1, String inputData2, HashMap<String, HashMap<String, String>> inputData3){
```

Comment:
Make parameter list variable name more meaningful, then you are good to go @chandx